docs/tutorials/podman_tutorial.md

```diff
@@ -140,10 +141,11 @@ podman ps
 Note: If you add *-a* to the *ps* command, Podman will show all containers.
 ### Inspecting a running container
 You can "inspect" a running container for metadata and details about itself.  We can even use
-the inspect subcommand to see what IP address was assigned to the container.
+the inspect subcommand to see what IP address was assigned to the container (none):
 ```console
 $ sudo podman inspect -l | grep IPAddress\":
-        "IPAddress": "10.88.6.140",
+            "SecondaryIPAddresses": null,
+            "IPAddress": "",
```

Comment:
this command runs as root.  Containers created by root get an IP

Reply:
Look again. The container in the tutorial gets started rootless and gets no IP. I have not changed that, just took it as it was.

Reply:
Need to kill the `sudo` then. We don't share containers between Root and Rootless Podman, so `sudo podman inspect` won't find a container created without using `sudo`

Reply:
Right, will do.